tests/IntegrationTests/SortingRequestTest.php

```diff
@@ -0,0 +1,72 @@
+<?php declare(strict_types=1);
+
+namespace Lmc\Matej\IntegrationTests;
+
+use Lmc\Matej\Model\Command\Interaction;
+use Lmc\Matej\Model\Command\UserMerge;
+
+/**
+ * @covers ItemPropertiesSetupRequestBuilder
+ * @covers EventsRequestBuilder
+ */
+class SortingRequestTest extends IntegrationTestCase
+{
+    /** @test */
+    public function shouldIssueSortingRequestOnly(): void
```

Comment:
What about use `execute` instead of `issue`? Its a bit confusing and uncommon verb for me...